cmd/acmesolver/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cert-manager/cert-manager/acmesolver-binary
 
-go 1.23.0
+go 1.24.0
+
+toolchain go1.24.9
```

Comment:
Not sure if we should include this line & what the consequences are, would prefer not having this line here.

Reply:
Ah, yeah, that's fair - I'll drop the toolchain directives!